wordSearch.c

```diff
@@ -55,7 +58,19 @@ int main(void)
             "<table style=font-family:arial;width15%%><tr><th "
             "colspan=2>Words</th></tr><tr>");
 
-    wordPick(fp, board);
+    FILE *wordFile = fopen(WORD_INPUT, "r");
+    if (!wordFile) {
+        fprintf(stderr, "Could not open %s\n", WORD_INPUT);
+        perror("fopen");
+        exit(1);
+    }
+
+    char words[FILE_WORD_COUNT][MAX_WORD_LENGTH] = {0};
+
+    loadWords(wordFile, words);
+
+    wordPick(fp, board, words);
+
```

Comment:
now the loading of words is done in `main()`, loaded once in the `wordFile` `FILE *` object, and then it is passed to `wordPick()`